src/core/tracking.js

```diff
@@ -107,6 +146,49 @@ export class Tracking {
       // (addons-frontend vs addons-server) is being used in analytics.
       ga('set', 'dimension3', 'addons-frontend');
     }
+
+    // Attempt to enable Hybrid content telemetry.
+    this.hctInitPromise = this.initHCT({ _config });
+  }
+
+  async initHCT({ _config = config } = {}) {
+    const hctEnabled = _config.get('hctEnabled');
+    if (typeof window !== 'undefined' && hctEnabled === true) {
+      log.info('Setting up the Hybrid Content Telemetry lib');
+      // Note: special webpack comments must be after the module name or
+      // babel-plugin-dynamic-import-node will blow-up.
+      try {
+        // prettier-ignore
+        const hybridContentTelemetry = await import(
+          'mozilla-hybrid-content-telemetry/HybridContentTelemetry-lib'
+          /* webpackChunkName: "disco-hct" */
+        );
+        await hybridContentTelemetry.initPromise();
```

Comment:
I noticed that when we do not have access to hct telemetry, this method stops here and there is no more logs, no errors, nothing..

Reply:
Hmm that would seem to be a bug in the lib. It should probably reject in this case.

Reply:
I've filed a bug https://bugzilla.mozilla.org/show_bug.cgi?id=1490284

Reply:
added: "blocked by upstream" label